app/jobs/destroy_collections_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# app/jobs/destroy_collections_job.rb
+class DestroyCollectionsJob < ApplicationJob
+  def perform
+    # Find specific collection
+    Collection.find_each do |collection|
+      next if collection.title.first == 'Journal of Adventist Education'
+
+      # Destroy the collections
+      collection.destroy!
+      puts "#{collection.title.first} was destroyed!"
```

Comment:
We may want to consider a Rails logger statement instead of a puts. We will likely be running this job in the background with DestroyCollectionsJob.perform_later so I think we would lose those messages. 

```suggestion
      Rails.logger.info "#{collection.title.first} was destroyed!"
```

Reply:
Done